aws/sdk/integration-tests/no-default-features/Cargo.toml

```diff
@@ -16,6 +16,7 @@ publish = false
 [dev-dependencies]
 aws-config = { path = "../../build/aws-sdk/sdk/aws-config", default-features = false }
 aws-sdk-s3 = { path = "../../build/aws-sdk/sdk/s3", default-features = false }
+aws-smithy-async = { path = "../../build/aws-sdk/sdk/aws-smithy-async", features = ["rt-tokio"] }
```

Comment:
Since this is specifically testing no default features, we should just create a no-op sleep impl for the test rather than pull in one of the default features. We don't actually need to sleep for anything in the test.

Reply:
Ah, got it. Will add a no-op sleep locally in the test and remove the `rt-tokio` feature. Thanks for pointing it out.

Reply:
Addressed in 45b91ab.